code/modules/reagents/chemistry/reagents/water.dm

```diff
@@ -461,3 +461,20 @@
 	reagent_state = LIQUID
 	color = "#29262b"
 	taste_description = "burnt dirt"
+
+/datum/reagent/tar_compound
+	name = "Sticky tar"
+	id = "sticky_tar"
+	description = "A sticky compound that creates tar on contact with surfaces."
+	reagent_state = LIQUID
+	color = "#4B4B4B"
+	harmless = FALSE
+	taste_description = "processed sludge"
+
+/datum/reagent/tar_compound/reaction_turf(turf/simulated/loc, volume)
```

Comment:
`loc` is a very common var name for ATOMs, could you just use `T` or something instead so it's not confusing?